modules/mono/editor/GodotTools/GodotTools.ProjectEditor/ProjectGenerator.cs

```diff
@@ -23,6 +23,8 @@ public static ProjectRootElement GenGameProject(string name)
 
             var mainGroup = root.AddPropertyGroup();
             mainGroup.AddProperty("TargetFramework", "net6.0");
+            var net70 = mainGroup.AddProperty("TargetFramework", "net7.0");
+            net70.Condition = " '$(GodotTargetPlatform)' == 'android' ";
```

Comment:
I'm not so sure about this change, this would only be added to newly created projects so users still need to modify their csproj manually in existing projects.